scripts/get_pool_status.py

```diff
@@ -3,18 +3,19 @@
 from __future__ import print_function
 import re
 import pprint
-import optparse
+import optparse  # pylint: disable=deprecated-module
 
-import classad
-import htcondor
+import htcondor2 as htcondor
```

Comment:
Did you also need `if 'useHtcV2' in os.environ:` for this file?

Reply:
we do not use this file anywhere. I am keeping it around as example just in case we need something like this in the future. Either way it will need testing and validation in case of any future use.